cmd/mpas/config/config.go

```diff
@@ -50,11 +50,15 @@ type BootstrapConfig struct {
 	Interval              string
 	CommitMessageAppendix string
 	Private               bool
+	Certificates          struct {
+		Name string
+		Dev  bool
```

Comment:
I find the dev & developer certs stuff confusing throughout;  might be just me though.
```suggestion
		GenerateSelfSigned  bool
```